.github/workflows/cicd-dev.yml

```
@@ -0,0 +1,78 @@
+name: MovieBookie CI/CD For Dev Pipeline
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)